opensrp-app/src/main/java/org/smartregister/SyncConfiguration.java

```diff
@@ -31,4 +31,7 @@ public boolean disableSyncToServerIfUserIsDisabled() {
         return false;
     }
 
+    public abstract SyncFilter getEncryptionParam();
+
+    public abstract boolean updateClientDetailsTable();
```

Comment:
Can't this method be implemented with a default value so that only implementations that want to override have to implement

Reply:
Yea, we want it to fail so that in the application level you get to choose the encryption param and whether to update the details table